src/core/Group.js

```diff
@@ -1793,7 +1793,7 @@ Phaser.Group.prototype.preUpdate = function ()
         return false;
     }
 
-    if (!this.exists || !this.parent.exists)
+    if (!this.active || !this.exists || !this.parent.exists)
```

Comment:
Would it make sense to check parent.active as well?

Reply:
The parents all do active checks on their children, so I think its overkill to go the opposite way.

But I might change that behaviour and then the parent .active check would make sense.

Reply:
Now **preUpdate** function calls won't be blocked by parents, and all **preUpdate** calls will call their **coreUpdate** even if they are not active so that they can destroy themselves again. **preUpdate** functions will now consider their parents as well.

**postUpdate**, **update** and **render** calls will all prune child calls through the active variable.